api/graphql/makeModels.js

```diff
@@ -191,7 +192,7 @@ export default async function makeModels (userId, isAdmin) {
         'created_at',
         'avatar_url',
         'banner_url',
-        'num_members',
+        'memberCount',
```

Comment:
I'm a bit confused why most attributes use underscores but postCount is is not. generally would prefer to stay consistent, but also not a big deal. 
also curious why you couldn't use the existing num_members? i do like having memberCount match the naming of postCount

Reply:
as discussed, the camel case is composed attributes and the snake case comes directly from the db